### PR TITLE
CHANGELOG: Document changes since v0.5.0

### PR DESCRIPTION
**Pull request by wking**
Through f7d6d292 (#806).  CC @abhinavdahiya, @crawford

### CODE REVIEW

**Comment by wking** [comment]
I've pushed dcc8077 -> eeca4c2, bumping the target release date to today.

/hold

There are a few minor changes I want to get in first (#793, #834, #840, and #849).  @abhinavdahiya, @crawford, can I get a `/lgtm` here?  I'd like to be able to land this once we get those PRs merged without anyone else having to stay up late ;).

**Comment by crawford** [comment]
Do we also want https://github.com/openshift/installer/pull/822?

**Comment by crawford** [comment]
Needs additional changes from 819ecb34843519ca71b113aca32e2cf1f85f55ac.

**Comment by wking** [comment]
I've pushed eeca4c2 -> 414b8c8, rebasing onto master now that #822 has landed.  I've picked up the Terraform entry from 819ecb3.  I've also added an additional `Unreleased` section so I can talk about #791, which has landed in master but which wasn't part of 0.6.0.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,59 @@ All notable changes to this project will be documented in this file.
...
+- We now push a `kubeadmin` user (with an internally-generated
+  password) into the cluster for the new [bootstrap identity
+  provider][bootstrap-identity-provider].  This gives users a way to
+  access the web console, Prometheus, etc. without needing to
+  configure a full-fledged identity provider or install `oc`.  The
+  `create cluster` subcommand now blocks until the web-console route
+  is available and then exits after printing instructions for using
+  the new credentials.
+- The installer binary now includes Terraform so there is no longer a
+  need to seperately download and install it.
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: `/s/seperately/separately`

Comment (wking) [reply CHANGELOG.md]: >     /s/seperately/separately

Fixed with 414b8c8 -> 05ba0fa, which also cleans up the section about #791.

**Comment by crawford** [comment]
+1 to the unreleased section. Let's encourage the team to write our own change log instead of relying on @wking.

**Comment by wking** [comment]
> +1 to the unreleased section. Let's encourage the team to write our own change log instead of relying on @wking 

The drawback to that is that it's a source of merge conflicts for otherwise orthogonal changes.  Middle ground is something like Kube's [fenced `release-note`][1], and I'd be fine moving to something like that.

[1]: https://github.com/kubernetes/kubernetes/blame/v1.13.0/.github/PULL_REQUEST_TEMPLATE.md#L30-L39

**Comment by wking** [comment]
Rebased onto master with 05ba0fa -> 08e4dbc, to document #817.  Now that we've emptied the LGTM backlog :tada:, I think we want to land this so we can start preparing for 0.7.0 :).

**Comment by wking** [comment]
[e2e-aws][1]:

```
level=fatal msg="Error executing openshift-install: waiting for bootstrap-complete: timed out waiting for the condition"
```

As discused in #857, that could be a number of things, but I haven't dug through the CI artifacts to try and pin it down.

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/841/pull-ci-openshift-installer-master-e2e-aws/2154/build-log.txt

**Pull request merged**